packages/node/src/channels.ts

```diff
@@ -80,9 +50,21 @@ export class Channels {
    */
   async createMultisig(params: Node.CreateMultisigParams): Promise<Address> {
     const multisigAddress = this.generateNewMultisigAddress(params.owners);
-    const channel: Channel = new Channel(multisigAddress, params.owners);
-    const ownersHash = orderedAddressesHash(params.owners);
-    await this.saveChannel(channel, ownersHash);
+    let stateChannel: StateChannel = new StateChannel(
```

Comment:
```suggestion
    let stateChannel = new StateChannel(
```

Reply:
This is also repeated in a few other places. It should be implicit from the return value of `StateChannel`'s constructor.